src/Features/CSharp/Portable/QuickInfo/OnTheFlyDocsUtilities.cs

```diff
@@ -33,6 +33,40 @@ internal static class OnTheFlyDocsUtilities
 
         OnTheFlyDocsRelevantFileInfo? GetOnTheFlyDocsRelevantFileInfo(ITypeSymbol typeSymbol)
         {
+            if (typeSymbol.IsTupleType && typeSymbol is INamedTypeSymbol tupleType)
+            {
+                foreach (var typeArgument in tupleType.TypeArguments)
+                {
+                    var elementInfo = GetOnTheFlyDocsRelevantFileInfo(typeArgument);
+                    if (elementInfo is not null)
+                    {
+                        return elementInfo;
+                    }
+                }
```

Comment:
whats' the general logic/thinking here?  I don't relaly get the idea of why we're just getting the first thing and returning, and ending with that.  for example, if i have `(string s, Customer c)` why do i only get relevant file info for `string` and nto `Customer`?